### ISSUE TITLE
Real Estate Price Prediction Model

### ISSUE THREAD

**Issue by amishhaa** [Akshat111111/Hedging-of-Financial-Derivatives#48]
I would like to build a Real Estate Price Prediction Model, if this enhancement aligns with the goal of this project can i go ahead and start working on this ?

**Comment by Akshat111111** [comment]
Yes you can work on it and can also compare the regions financial market growth increase with the financial index increase

**Comment by shikham12** [comment]
can i work on this as well?

**Comment by Pranshu-jais** [comment]
Approach :  Creating a Real Estate Price Prediction Model  using ml algorithm like XGBoost,Random Forest,Lasso Regression etc or a deep learning model trained on  relevant features that significantly impact real estate prices . i request to assign me this issue   with the gssoc label and the appropriate level (1/2/3).

**Comment by thakurboy004** [comment]
I would like to contribute to this issue, and one more thing please mark this issue with appropriate tags to GSSoC'24 contributors

**Comment by Akshat111111** [comment]
> Approach : Creating a Real Estate Price Prediction Model using ml algorithm like XGBoost,Random Forest,Lasso Regression etc or a deep learning model trained on relevant features that significantly impact real estate prices . i request to assign me this issue with the gssoc label and the appropriate level (1/2/3).

Done,You can move forward

**Comment by Akshat111111** [comment]
> I would like to contribute to this issue, and one more thing please mark this issue with appropriate tags to GSSoC'24 contributors

Tell me about your approach ??

**Comment by Akshat111111** [comment]
> I would like to contribute to this issue, and one more thing please mark this issue with appropriate tags to GSSoC'24 contributors

Approach ??

**Comment by amishhaa** [comment]
@Akshat111111 I created this issue first and someone else is assigned to work on this now which is causing a lot of confusion for me. Can you please clarify if I am still supposed to work on this regardless of the assignment?

**Comment by thakurboy004** [comment]
> > I would like to contribute to this issue, and one more thing please mark this issue with appropriate tags to GSSoC'24 contributors
> 
> Tell me about your approach ??

Approach: I will be applying Deep Neural Network (DNN) to build a highly accurate model for prediction and also performing appropriate data preprocessing to keep the model as optimized as I can

**Comment by Akshat111111** [comment]
> @Akshat111111 I created this issue first and someone else is assigned to work on this now which is causing a lot of confusion for me. Can you please clarify if I am still supposed to work on this regardless of the assignment?

You can work on the idea that I have suggested

**Comment by Akshat111111** [comment]
> > > I would like to contribute to this issue, and one more thing please mark this issue with appropriate tags to GSSoC'24 contributors
> > 
> > 
> > Tell me about your approach ??
> 
> Approach: I will be applying Deep Neural Network (DNN) to build a highly accurate model for prediction and also performing appropriate data preprocessing to keep the model as optimized as I can

I will suggest to craete a new issue and start working.

**Comment by amishhaa** [comment]
> > @Akshat111111 I created this issue first and someone else is assigned to work on this now which is causing a lot of confusion for me. Can you please clarify if I am still supposed to work on this regardless of the assignment?
> 
> You can work on the idea that I have suggested

thank you for clarifying i will shortly create a new issue and PR with my proposed model for the topic regions financial market growth increase with the financial index increase.

**Comment by Akshat111111** [comment]
sure

**Comment by keshavk21** [comment]
May I work on it??

**Comment by Akshat111111** [comment]
Kindly tell your approach?

**Comment by aryanchauh** [comment]
Can u assign me this issue @Akshat111111
